File: sandbox/python/src/FaxLineAreaCodeGetExample.py
import json
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    username="YOUR_API_KEY",
)

with ApiClient(configuration) as api_client:
    try:
        response = api.FaxLineApi(api_client).fax_line_area_code_get(
            country="US",
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling FaxLineApi#fax_line_area_code_get: %s\n" % e)

File: sandbox/node/src/FaxLineAreaCodeGetExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.FaxLineApi();
apiCaller.username = "YOUR_API_KEY";

apiCaller.faxLineAreaCodeGet(
  "US", // country
  undefined, // state
  undefined, // province
  undefined, // city
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling FaxLineApi#faxLineAreaCodeGet:");
  console.log(error.body);
});
